auth0/v3/management/grants.py

```diff
@@ -0,0 +1,65 @@
+from .rest import RestClient
+
+
+class Grants(object):
+
+    """Auth0 grants endpoints
+
+    Args:
+        domain (str): Your Auth0 domain, e.g: 'username.auth0.com'
+
+        token (str): Management API v2 Token
+
+        telemetry (bool, optional): Enable or disable Telemetry
+            (defaults to True)
+    """
+
+    def __init__(self, domain, token, telemetry=True):
+        self.domain = domain
+        self.client = RestClient(jwt=token, telemetry=telemetry)
+
+    def _url(self, id=None):
+        url = 'https://%s/api/v2/grants' % self.domain
+        if id is not None:
+            return url + '/' + id
+        return url
+
+    def get_all(self, user_id=None, client_id=None, audience=None, page=None, per_page=None, include_totals=False):
```

Comment:
This is typically called `all` (see above for examples)

Reply:
Also .... this is a lot of parameters that aren't required. Can we pass in `options` here?